pkg/kubefed2/federate/enable.go

```diff
@@ -253,6 +253,10 @@ func typeConfigForTarget(apiResource metav1.APIResource, enableTypeDirective *En
 	spec := enableTypeDirective.Spec
 	kind := apiResource.Kind
 	pluralName := apiResource.Name
+	shortNames := []string{}
+	for _, shortName := range apiResource.ShortNames {
+		shortNames = append(shortNames, fmt.Sprintf("f%s", shortName))
```

Comment:
Is `f<short name>` sufficiently descriptive, or would a lengthier prefix like 'fed' be desirable?

(No action required) Not required for this PR, but it may be desirable in future to support configuring either the prefix or the short name for an enable invocation.